restapi/src/main/java/io/stargate/web/impl/RestApiRunner.java

```diff
@@ -21,18 +21,14 @@
 import io.stargate.core.metrics.api.Metrics;
 import io.stargate.db.datastore.DataStoreFactory;
 
-public class WebImpl {
+public class RestApiRunner {
 
   private AuthenticationService authenticationService;
   private AuthorizationService authorizationService;
   private Metrics metrics;
   private HttpMetricsTagProvider httpMetricsTagProvider;
   private DataStoreFactory dataStoreFactory;
 
-  public AuthenticationService getAuthenticationService() {
```

Comment:
As far as I know, no getters are needed. Could also just remove setters as well, pass via constructor.

Reply:
Or even go for the immutable class..